### PR TITLE
Feature: User tests

### PR DESCRIPTION
**Pull request by agerard57**

### CODE REVIEW

File: api/test/index.js [hunk 1]
```diff
@@ -1,4 +1,68 @@
...
-testList();
\ No newline at end of file
+const mainTests = async () => {
+
+  const mainUser = {
+    first_name:"Toby",
+    last_name: "toto",
+    email: "tutu@gmail.com",
+    password: "1234",
+    color: "#873260" ,
```
Comment (agerard57) [inline api/test/index.js]: Use faker

File: api/test/index.js [hunk 2]
```diff
@@ -1,4 +1,68 @@
...
+          body: res.json(),
+          status: res.status,
+        })
+    );
+  };
+
+  const getToken = async () => {
+    createUser();
+  
+    const raiponce = await fetch(`${baseUrl}/user/login`, {
```
Comment (agerard57) [inline api/test/index.js]: Shame.

File: api/test/index.js [hunk 3]
```diff
@@ -1,4 +1,68 @@
...
+      method: 'POST',
+      body: JSON.stringify({
+        email: mainUser.email,
+        password: mainUser.password,
+      }),
+    });
+    const vaiana = await raiponce.json();
+  
+    tiana = vaiana.token;
+    return tiana;
```
Comment (agerard57) [inline api/test/index.js]: This too. Pure filth

File: client/src/core/components/ColorVignette.tsx [hunk 4]
```diff
@@ -3,12 +3,12 @@ import { css } from '@emotion/react';
 import { FC } from 'react';
 
 type Props = {
-  firstName: string;
+  first_name: string;
   color: string;
   size: number;
```
Comment (agerard57) [inline client/src/core/components/ColorVignette.tsx]: Revert

File: client/src/core/components/ColorVignette.tsx [hunk 5]
```diff
@@ -3,12 +3,12 @@ import { css } from '@emotion/react';
...
 };
 
-export const ColorVignette: FC<Props> = ({ firstName, color, size }) => (
```
Comment (agerard57) [inline client/src/core/components/ColorVignette.tsx]: Revert

File: client/src/core/components/kanban/KanbanCard.tsx [hunk 6]
```diff
@@ -63,7 +63,7 @@ export const KanbanCard: FC<CardProps> = ({ card }) => {
           <ColorVignette
             color={card.author.color}
             size={30}
-            firstName={card.author.name.firstName}
```
Comment (agerard57) [inline client/src/core/components/kanban/KanbanCard.tsx]: Revert

File: client/src/core/types/User.ts [hunk 7]
```diff
@@ -1,7 +1,7 @@
 export interface PasswordlessUser {
   id: number;
   name: {
-    firstName: string;
```
Comment (agerard57) [inline client/src/core/types/User.ts]: Revert

File: client/src/landingPage/components/LandingPage.tsx [hunk 8]
```diff
@@ -17,7 +17,7 @@ export const LandingPage: FC = () => {
   const author1: PasswordlessUser = {
     id: 1,
     name: {
-      firstName: 'Alexandre',
```
Comment (agerard57) [inline client/src/landingPage/components/LandingPage.tsx]: Revert

File: client/src/landingPage/components/LandingPage.tsx [hunk 9]
```diff
@@ -27,7 +27,7 @@ export const LandingPage: FC = () => {
   const author2: PasswordlessUser = {
     id: 2,
     name: {
-      firstName: 'Nicolas',
```
Comment (agerard57) [inline client/src/landingPage/components/LandingPage.tsx]: Revert

File: client/src/landingPage/components/UserList.tsx [hunk 10]
```diff
@@ -37,7 +37,7 @@ export const UserList: FC<Props> = ({ lists }) => (
           color={author.color}
           key={author.id}
           size={40}
-          firstName={`${author.name.firstName} ${author.name.lastName}`}
```
Comment (agerard57) [inline client/src/landingPage/components/UserList.tsx]: Revert

File: docs/KanBan.postman_collection.json [hunk 11]
```diff
@@ -278,7 +278,7 @@
 						"header": [],
 						"body": {
 							"mode": "raw",
-							"raw": "{\r\n    \"firstname\" : \"vincent\",\r\n    \"surname\" : \"remy\",\r\n    \"email\" : \"rsssssssssssey@gml.com\",\r\n    \"password\" : \"Azerty123\",\r\n    \"color\" : \"#ff0000\"\r\n}",
```
Comment (agerard57) [inline docs/KanBan.postman_collection.json:281]: I don't know whether if this is camel or snake, please verify

File: docs/KanBan.postman_collection.json [hunk 12]
```diff
@@ -381,7 +381,7 @@
 						"header": [],
 						"body": {
 							"mode": "raw",
-							"raw": "{\r\n    \"surname\" : \"REMY\"\r\n}",
+							"raw": "{\r\n    \"last_name\" : \"REMY\"\r\n}",
```
Comment (agerard57) [inline docs/KanBan.postman_collection.json]: Same as above.

**Review by agerard57 — COMMENTED** [review]
Nope

**Review by Virex22 — COMMENTED** [review]
up,
please fix asap

**Pull request merged**